psalm-baseline.xml

```diff
@@ -363,6 +372,9 @@
       <code>onExtract</code>
       <code>onHydrate</code>
     </InternalMethod>
+    <DeprecatedMethod occurrences="5">
+      <code>$this->returnValue</code>
+    </DeprecatedMethod>
```

Comment:
These deprecations can be fixed in the test with `MockObject::willReturn($whatever)`